sysboundary/setmaxwellian.cpp

```diff
@@ -143,14 +143,17 @@ namespace SBC {
       
       const vmesh::LocalID* vblocks_ini = cell.get_velocity_grid_length(popID,refLevel);
 
+      const Real dvx=cell.get_velocity_grid_cell_size(popID,refLevel)[0];
+      const Real dvy=cell.get_velocity_grid_cell_size(popID,refLevel)[1];
+      const Real dvz=cell.get_velocity_grid_cell_size(popID,refLevel)[2];
       while (search) {
          #warning TODO: add SpatialCell::getVelocityBlockMinValue() in place of sparseMinValue?
          if (0.1 * getObjectWrapper().particleSpecies[popID].sparseMinValue > 
              maxwellianDistribution(
                                     popID,
                                     rho,
                                     T,
-                                    VX0 + counter*cell.get_velocity_grid_block_size(popID,refLevel)[0], VY0, VZ0
+                                    counter*cell.get_velocity_grid_block_size(popID,refLevel)[0] +0.5*dvx , 0.5*dvy, 0.5*dvz
```

Comment:
I have a question here. `dvy` and `dvz` are supposed to be the vgrid cell size in y and z dimensions, while in the original form `VY0` and `VZ0`  are the centered velocities in y and z. Is it correct to replace `VY0, VZ0` with `0.5*dvy` and `0.5*dvz`?

Reply:
So this means "a Maxwellian distribution centered at 0 with width dv"? Does it also mean that with the current dev branch, if I impose a nonzero y/z velocity component, then it will be completely off?

Reply:
Yep, you got it. So? Big fat comment to explain this to future-us, or a different approach?